GANDLF/entrypoints/debug_info.py

```diff
@@ -37,6 +51,20 @@ def new_way():
     + "`gandlf_debugInfo` script would be deprecated soon."
 )
 def old_way():
+    parser = argparse.ArgumentParser(
+        prog="GANDLF_DebugInfo",
+        formatter_class=argparse.RawTextHelpFormatter,
+        description="Generate debugging information for maintainers.\n\n"
+        + copyrightMessage,
+    )
+    parser.add_argument(
+        "-v",
+        "--verbose",
+        metavar="",
+        type=ast.literal_eval,
+        default=False,
```

Comment:
I'd propose to use `parser.add_argument('--verbose', '-v', default=False, action='store_true')` still as it is more standard way to handle boolean flags. Indeed, in other old-way commands we parse the scripts with `type=ast.literal_eval` but that's a really broken and error-prone way:)

Reply:
Fixed.